codec/src/types/hash_set.rs

```diff
@@ -306,7 +306,7 @@ mod tests {
         2u8.write(&mut expected2); // Item 2
         5u8.write(&mut expected2); // Item 5
         assert_eq!(set2.encode(), expected2.freeze());
-        assert_eq!(set2.encode_size(), 1 + 3 * u8::SIZE);
+        assert_eq!(set2.encode_size(), 1 + 3);
```

Comment:
Not thrilled that this fires a warning now :/

Maybe we `#[allow(clippy::identity_op)]` here or just disable that lint globally?

Reply:
discussed offline; keeping given it's in a test.